### PR TITLE
MGMT-16871 Highlight currently selected nav entry

### PR DESCRIPTION
**Pull request by celdrake**
https://issues.redhat.com/browse/MGMT-16871

Highlights the entries of the menu based on the current location.

Updated 16/Feb: Since we're using `NavLink` from React Router, these know when they are active. But they render as links, and we cannot use them to wrap the NavItems,. So I used CSS to detect the item that has the `active` class set by React router. It seems to work well for all nested routes, etc.

### CODE REVIEW

**Comment by rawagner** [comment]
If I go to details (ie Fleet Details), the Fleet NavItem stops being highlighted

File: src/app/components/AppLayout/AppToolbar.css [hunk 1]
```diff
@@ -1,3 +1,16 @@
...
+  - Done by detecting the "active" class set by React Router on the active Route
+  - Highlights the entry for the active Route
+  - Highlights the section that includes the active Route, but only when it's collapsed
+ */
+.pf-v5-c-nav__item:not(.pf-m-expanded):has(.pf-v5-c-nav__link.active) button {
+  background-color: var(--pf-v5-c-nav__link--active--BackgroundColor);
+}
+.pf-v5-c-nav__item:not(.pf-m-expandable):has(.pf-v5-c-nav__link.active) {
+  background-color: var(--pf-v5-c-nav__link--active--BackgroundColor);
+}
```
Comment (rawagner) [inline src/app/components/AppLayout/AppToolbar.css:16]: These should go to `AppNavigation.css` file


We should also scope these css changes to under some `fctl` component to make sure we wont override some other parts of the app (especially when we will integrate into other products..even though, our Navigation wont appear there..but still, i'd just make sure to scope these changes)

**Review by rawagner — CHANGES_REQUESTED** [review]
works fine!
Lets address my concerns about the css rules & location before merging

File: src/app/components/AppLayout/AppToolbar.css (hunk 1 above)
Comment (celdrake) [reply src/app/components/AppLayout/AppToolbar.css:16]: Good suggestions, I agree with both. I'll update the MR

**Review by rawagner — APPROVED** [review]

**Pull request merged**